databases/week2/task_3_more_queries.sql

```diff
@@ -0,0 +1,43 @@
+-- Part 3: More queries
+USE hyf_lesson2;
+
+-- 3.1 Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT t.id, t.title, t.description, t.created, t.updated, t.due_date, t.status_id 
+FROM user_task ut 
+INNER JOIN task t ON ut.task_id=t.id 
+WHERE ut.user_id = (
+    SELECT id
+    FROM user 
+    WHERE email LIKE '%@spotify.com'
+);
+
+-- 2. Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT t.title
+FROM task t
+INNER JOIN user_task ut ON t.id = ut.task_id
+INNER JOIN status s ON t.status_id = s.id 
+WHERE ut.user_id = (
+    SELECT id 
+    FROM user
+    WHERE name = 'Donald Duck'
+)
+AND s.id = (
+    SELECT id
+    from status
+    WHERE name = 'Not started'
+);	
+
```

Comment:
This solution works okay in this case
Now please think about this.
What would you do if your sub selects return more than one row?
```

SELECT t.title
FROM task t
INNER JOIN user_task ut ON t.id = ut.task_id
INNER JOIN status s ON t.status_id = s.id 
WHERE ut.user_id **IN** (
    SELECT id 
    FROM user
    WHERE name = 'Donald Duck'
)
AND s.id **IN** (
    SELECT id
    from status
    WHERE name = 'Not started'
);
```